burst/providers/providers.json

```diff
@@ -3694,5 +3694,48 @@
         "tv_extra2": "",
         "tv_keywords": "{title:original:en:hu} s{season:2}e{episode:2}",
         "tv_keywords_fallback": "{title:original:en:hu} s{season:2}"
+    },
+    "torrentio": {
+      "enabled": true,
+      "is_api":true,
+      "predefined": true,
+      "name": "Torrentio",
+      "private": false,
+      "languages": "pt",
+      "anime_extra": "stream/series/{imdb_id}:{season}:{episode}.json",
+      "anime_keywords": "{title:original:en} {year}",
+      "anime_keywords_fallback": "",
+      "anime_keywords_fallback2": "",
+      "anime_keywords_fallback3": "",
+      "anime_query": "",
+      "base_url": "https://torrentio.strem.fun/providers=yts,eztv,rarbg,1337x,thepiratebay,kickasstorrents,torrentgalaxy,magnetdl,horriblesubs,nyaasi,comando,bludv|sort=qualitysize|qualityfilter=480p,scr,cam/EXTRA",
+      "color": true,
+      "general_extra": "asdasd",
+      "general_keywords": "{title:original}",
+      "general_query": "",
+      "language": "pt",
+      "login_failed": "",
+      "login_object": "",
+      "login_path": null,
+      "movie_extra": "stream/movies/{imdb_id}.json",
+      "movie_keywords": "{title:original:en} {year}",
+      "movie_query": "",
+      "download_path": "magnet:?xt=urn:btih:",
+      "api_format": {
+        "name": "title",
+        "info_hash": "infoHash",
+        "results": "streams",
+        "torrent": "infoHash"
+      },
+      "season_extra": "",
+      "season_extra2": "",
+      "season_keywords": "{title:original:en} s{season:2}",
+      "season_keywords2": "{title:original:en} season {season}",
+      "season_keywords_fallback": "{title:original:en} {season:2}x01",
+      "season_query": "",
+      "separator": "+",
+      "subpage": false,
+      "tv_extra": "stream/series/{imdb_id}:{season}:{episode}.json",
+      "tv_keywords": "{title:original:en:hu} s{season:2}e{episode:2}"
```

Comment:
Hungarian language as the fallback?

@Rodrigo-Barros Let's replace languages and remove `hu` here.